templates/trusted-node/cdk-node-config.toml

```diff
@@ -31,8 +31,9 @@ MaxPendingTx = 1
 {{if eq .zkevm_rollup_fork_id "12"}}
 MaxBatchesForL1 = 300
 BlockFinality="SafeBlock" # TODO: should be finalized but it breaks :)
-SanityCheckRPCURL = ""
```

Comment:
Don't we need this parameter set? https://github.com/0xPolygon/cdk/blob/591e4017d78596ed420acb8d45e2b91758136e33/sequencesender/config.go#L71

Reply:
It is not a sanity check anymore, but a required parameter.